java/java-tests/testSrc/com/intellij/psi/formatter/java/JavaFormatterAlignmentTest.java

```diff
@@ -49,6 +49,20 @@ public void testMethodAndChainedField() throws Exception {
     );
   }
 
+  public void testChainedMethodWithComments() throws Exception {
+    getSettings().ALIGN_MULTILINE_CHAINED_METHODS = true;
+    doMethodTest("AA.bb()\n" +
+                 ".cc() // comment after line\n" +
+                 ".dd()\n " +
+                 "        /* block comment on empty line */\n" +
```

Comment:
We could, but I think conceptually - and even for the fix - they ended up being exactly the same, don't see the added value in separating them.

Let me know if you want me to add it anyway.

(FYI this is related to https://github.com/JetBrains/intellij-community/pull/298)